ide/api.lsp/src/org/netbeans/spi/lsp/ErrorProvider.java

```diff
@@ -71,9 +72,32 @@ public Context(FileObject file, Kind errorKind) {
          * @since 1.4
          */
         public Context(FileObject file, int offset, Kind errorKind) {
+            this(file, offset, errorKind, null);
+        }
+
+        /**
+         * Construct a new {@code Context}.
+         *
+         * @param file file for which the errors/warnings should be computed
+         * @param offset offset for which the errors/warnings should be computed
+         * @param errorKind the type of errors/warnings that should be computed
+         * @param hintsConfigFile file which contains preferences for the the errors/warnings to be computed
+         *
+         */
```

Comment:
Some additional changes are needed to properly handle an API addition:
- the version of the module should be increased, in this case the `OpenIDE-Module-Specification-Version` in `api.lsp/manifest.mf` should be incremented.
- the new API elements should have `@since ...` referring to the new version
- `ide/api.lsp/apichanges.xml` should be enhanced with the description of the change.